src/ado2gh/Commands/DownloadLogsCommand.cs

```diff
@@ -127,21 +136,36 @@ HttpDownloadService httpDownloadService
                 _log.LogWarning($"Overwriting {migrationLogFile} due to --overwrite option.");
             }
 
-            _log.LogInformation($"Downloading log for repository {githubRepo} to {migrationLogFile}...");
-
             var githubApi = _githubApiFactory.Create(githubApiUrl, githubPat);
-            var logUrl = await githubApi.GetMigrationLogUrl(githubOrg, githubRepo);
+            string logUrl;
 
-            if (logUrl == null)
+            while (true)
             {
-                throw new OctoshiftCliException($"Migration not found for repository {githubRepo}!");
-            }
+                logUrl = await githubApi.GetMigrationLogUrl(githubOrg, githubRepo);
```

Comment:
nit: there is a syntactic sugar to throw an exception if the returned value is null:
```suggestion
                logUrl = await githubApi.GetMigrationLogUrl(githubOrg, githubRepo) ?? throw new OctoshiftCliException($"Migration not found for repository {githubRepo}");
```

Reply:
Updated in 1a2da58aa27d4397aa73600386cb23f500637225!